### ISSUE TITLE
['Win10', 'Py3.72', 'pipx-0.12.3.1'] FileNotFoundError when installing package with deps without binary

### ISSUE THREAD

**Issue by Laurentiu-Andronache** [pipxproject/pipx#98]
```
C:\Projects\Python\_helping\pipx>pipx list
nothing has been installed with pipx

C:\Projects\Python\_helping\pipx>pipx install --include-deps tox-venv
Traceback (most recent call last):
  File "C:\Python37\Scripts\pipx-script.py", line 11, in <module>
    load_entry_point('pipx', 'console_scripts', 'pipx')()
  File "c:\projects\python\_helping\pipx\pipx\main.py", line 496, in cli
    exit(run_pipx_command(parsed_pipx_args, binary_args))
  File "c:\projects\python\_helping\pipx\pipx\main.py", line 157, in run_pipx_command
    include_deps=args.include_deps,
  File "c:\projects\python\_helping\pipx\pipx\commands.py", line 319, in install
    _run_post_install_actions(venv, package, local_bin_dir, venv_dir, include_deps)
  File "c:\projects\python\_helping\pipx\pipx\commands.py", line 345, in _run_post_install_actions
    _expose_binaries_globally(local_bin_dir, binary_paths, package)
  File "c:\projects\python\_helping\pipx\pipx\commands.py", line 470, in _expose_binaries_globally
    _copy_package_binaries(local_bin_dir, binary_paths, package)
  File "c:\projects\python\_helping\pipx\pipx\commands.py", line 485, in _copy_package_binaries
    shutil.copy(src, dest)
  File "c:\python37\lib\shutil.py", line 245, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "c:\python37\lib\shutil.py", line 120, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Nebu\\.local\\pipx\\venvs\\tox-venv\\Scripts\\tox'

C:\Projects\Python\_helping\pipx>pipx install --include-deps tox-venv
Not installing to existing directory 'C:\\Users\\Nebu\\.local\\pipx\\venvs\\tox-venv'. Pass '--force' to force installation
```

Added what's hopefully the solution to my pull request: https://github.com/pipxproject/pipx/pull/97

**Comment by cs01** [comment]
Thanks! For future reference you can put "#97" in the text of the commit message and GitHub will automatically add a link to this issue.

**Comment by cs01** [comment]
Oh sorry I should have mentioned, not a big deal but you generally want to put a summary followed by the issue number. Something like "fix FileNotFoundError on windows (#97)".

**Comment by Laurentiu-Andronache** [comment]
Got it! Thanks for all these tips.

**Comment by Laurentiu-Andronache** [comment]
Btw, I think that my fix might be a bit shallow. Meaning that it does make it so that installing in this case works, but it doesn't solve the actual issue of why pipx tries to copy a file that doesn't exist in the first place.

**Comment by cs01** [comment]
This has been fixed in 0.12.3.2
